fractions/javaee/remoting/src/main/java/org/wildfly/swarm/remoting/runtime/RemotingLegacyConnectorCustomizer.java

```diff
@@ -40,7 +40,7 @@
     @Override
     public void customize() {
         if (this.remoting.isRequireLegacyConnector() ) {
-            LOG.info("Remoting installed but Undertnow not available. Enabled legacy connector on port 4777.");
+            LOG.info("Remoting installed but Undertnow not available. Enabled legacy connector on port 4447.");
```

Comment:
I didn't notice it when looking at the code, and it's totally unrelated to this PR, but this line has a typo in the word Undertow :-)